src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -437,42 +432,52 @@ def _build_for_jumpstart(self):
         self.secret_key = None
         self.jumpstart = True
 
-        pysdk_model = self._create_pre_trained_js_model()
+        self.pysdk_model = self._create_pre_trained_js_model()
+        self.pysdk_model.tune = lambda *args, **kwargs: self._default_tune()
 
-        image_uri = pysdk_model.image_uri
-
-        logger.info("JumpStart ID %s is packaged with Image URI: %s", self.model, image_uri)
+        logger.info(
+            "JumpStart ID %s is packaged with Image URI: %s", self.model, self.pysdk_model.image_uri
+        )
 
-        if self._is_gated_model(pysdk_model) and self.mode != Mode.SAGEMAKER_ENDPOINT:
-            raise ValueError(
-                "JumpStart Gated Models are only supported in SAGEMAKER_ENDPOINT mode."
-            )
+        if self.mode != Mode.SAGEMAKER_ENDPOINT:
+            if self._is_gated_model(self.pysdk_model):
+                raise ValueError(
+                    "JumpStart Gated Models are only supported in SAGEMAKER_ENDPOINT mode."
+                )
 
-        if "djl-inference" in image_uri:
-            logger.info("Building for DJL JumpStart Model ID...")
-            self.model_server = ModelServer.DJL_SERVING
+            if "djl-inference" in self.pysdk_model.image_uri:
```

Comment:
can we add `huggingface-pytorch-inference` in the image URI matching logic and map ti model server of MMS.

Reply:
Let's add it next as we may need to add tune capability for it.

Reply:
no tune function needed for the HF container, the goal is to make it work with local container mode first.